src/Components/Server/src/Circuits/Circuit.cs

```diff
@@ -1,35 +1,23 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using Microsoft.JSInterop;
-
 namespace Microsoft.AspNetCore.Components.Server.Circuits
 {
     /// <summary>
-    /// Represents an active connection between a Blazor server and a client.
+    /// Represents a link between a ASP.NET Core Component on the server and a client.
     /// </summary>
-    public class Circuit
+    public sealed class Circuit
```

Comment:
What's the purpose of this class? It only has a single ID and a _circuitHost that is not accessible outside this class. Can't the Id be moved to the CircuitHost class and this class be removed?

Reply:
I expect `Circuit` to be the user facing type. `CircuitHost` is an implementation detail.